mbed_targets/mbed_targets.py

```diff
@@ -123,12 +121,20 @@ class MbedTargets(Set):
 
     @classmethod
     def from_offline_database(cls) -> "MbedTargets":
-        """Initialise with the offline target database."""
+        """Initialise with the offline target database.
+
+        Raises:
+            TargetDatabaseError: Could not retrieve data from the target database.
```

Comment:
When does it raise this error?

Reply:
Raised in the call to `get_*_database` although I think it's relevant to the API as users are exposed to it, so it should have some documentation.

Reply:
I'm following now. 👍